queries/lrb/src/main/java/storm/lrb/tools/Helper.java

```diff
@@ -44,5 +44,7 @@ public static String readable(List<String> fields) {
 		return tmp.toString();
 	}
 	
+	public final static String ISSUE_REPORT_URL = "https://github.com/mjsax/aeolus/issues/new";
```

Comment:
just some constant I use in runtime assertions in error messages like "bla bla failed, please report at bug at ..."